src/main/java/appeng/helpers/LocationRotation.java

```diff
@@ -28,10 +28,10 @@
 public class LocationRotation implements IOrientable
 {
 
-	final IBlockAccess w;
-	final int x;
-	final int y;
-	final int z;
+	private final IBlockAccess w;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused "w" private field. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AS1068)